src/interfaces/agm-ext-eval-interface.ts

```diff
@@ -21,5 +21,5 @@ export const EmptyAgmExtEval: AgmExtEvalIF = {
   agmYear: null,
   extensionDuration: NaN,
   agmDueDate: null,
-  isEligible: false
+  isEligible: null
```

Comment:
Do you have any state variables that belong in here (and that we should save to the filing JSON for audit purposes)?

Reply:
Nothing that comes to mind as of right now. Also, I reverted this change to false Sev.

Reply:
OK.

FYI, between all our PRs, I'm trying to determine which of these object properties are optional or not. I think there's probably little gain into making any of them optional (depending on what the filing JSON will look like), fwiw. No change needed here; I can do it in my "pull it all together" ticket.

Reply:
Let me know if you have any thoughts about the above. (You, too, Leo and Ketaki.)

Reply:
I think @PaulGarewal expects a couple of computed properties for his component.

Leo and Paul, can you work through this together, please?